features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScopeTest.kt

```diff
@@ -7594,6 +7622,165 @@ internal class RumViewScopeTest {
 
     // endregion
 
+    // region Feature Flags Batch
+
+    @Test
+    fun `M send event W handleEvent(AddFeatureFlagBatchEvaluation) on active view`(
```

Comment:
class name should be updated here and in the tests below

```suggestion
    fun `M send event W handleEvent(AddFeatureFlagEvaluations) on active view`(
```